processor/src/operations/sys_ops.rs

```diff
@@ -92,6 +92,19 @@ where
 
         Ok(())
     }
+
+    // CLOCK CYCLE
+    // --------------------------------------------------------------------------------------------
+
+    /// Pushes the current clock cycle onto the stack. The clock cycle is the number of times the
+    /// `execute` method has been called on the VM. The clock cycle is incremented after the
+    /// execution of each operation.
```

Comment:
I think this isn't quite right. The clock counter is incremented for every `execute_op` call, which is much more than every time `execute` is called.

Reply:
I've changed it to `execute_op`.